src/components/Pagination/docs/PaginationProvider.tsx

```diff
@@ -7,54 +7,77 @@ interface Props {
     children: (props: {
         currentPage: number;
         pageSize: number;
+        pageSizes: { label: string; value: string }[];
         totalItems: number;
         handleNextPage: () => void;
         handlePrevPage: () => void;
+        handleSelectPageSize: (selected: { label: string; value: string }) => void;
         handleSkipForward: () => void;
         handleSkipBackward: () => void;
     }) => ReactNode;
 }
 
 interface State {
     currentPage: number;
+    pageSize: number;
 }
 
 class PaginationProvider extends React.Component<Props, State> {
```

Comment:
I know it's an example, but can we take advantage of the changes to make it a function?  😊

Reply:
Yes sure, I'll have it done by this afternoon

Reply:
Done 👍🏽